lib/jsdom/living/nodes/GlobalEventHandlers-impl.js

```diff
@@ -1,6 +1,202 @@
 "use strict";
 
-class GlobalEventHandlersImpl { }
+const vm = require("vm");
+
+const idlUtils = require("../generated/utils");
+
+const ErrorEvent = require("../generated/ErrorEvent");
+
+const events = new Set([
+  "abort", "autocomplete",
+  "autocompleteerror", "blur",
+  "cancel", "canplay", "canplaythrough",
+  "change", "click",
+  "close", "contextmenu",
+  "cuechange", "dblclick",
+  "drag", "dragend",
+  "dragenter", "dragexit",
+  "dragleave", "dragover",
+  "dragstart", "drop",
+  "durationchange", "emptied",
+  "ended", "error", "focus",
+  "input", "invalid",
+  "keydown", "keypress",
+  "keyup", "load", "loadeddata",
+  "loadedmetadata", "loadstart",
+  "mousedown", "mouseenter",
+  "mouseleave", "mousemove",
+  "mouseout", "mouseover",
+  "mouseup", "wheel",
+  "pause", "play",
+  "playing", "progress",
+  "ratechange", "reset",
+  "resize", "scroll",
+  "seeked", "seeking",
+  "select", "show",
+  "sort", "stalled",
+  "submit", "suspend",
+  "timeupdate", "toggle",
+  "volumechange", "waiting"
+]);
+
+const inheritedAttributes = new Set(["blur", "error", "focus", "load", "resize", "scroll"]);
```

Comment:
[They are](https://html.spec.whatwg.org/#event-handler-content-attributes), "kind of".